src/components/experiment-tracking/experiment-primary-toolbar/experiment-primary-toolbar.js

```diff
@@ -60,6 +70,14 @@ export const ExperimentPrimaryToolbar = ({
         visible={enableComparisonView}
         disabled={showChangesIconDisabled}
       />
+      <CSVLink data={exportData} filename={generateCsvFileName(runMetadata)}>
```

Comment:
Pedantic suggestion: this function name could use all capitals for CSV to match the `CSVLink` component naming.

```suggestion
      <CSVLink data={exportData} filename={generateCSVFileName(runMetadata)}>
```